openquakeplatform/openquakeplatform/static/js/risk_input_model_exposure.js

```diff
@@ -0,0 +1,437 @@
+        } else if (limitState == 'relative') {
+            insuranceLimit = '\t\t\t<insuranceLimit isAbsolute="false"/>\n';
+            limitValue = ' insuranceLimit="'+data[i][limitInx]+'"';
+        }
+
+        // deductibleSelect
+        var deductibleValue = '';
+        var deductibleState = $('#deductibleSelect option:selected').val();
+        if (deductibleState == 'absolute') {
+            deductible = '\t\t\t<deductible isAbsolute="true"/>\n';
+            deductibleValue = ' deductible="'+data[i][deductibleInx]+'"';
+        } else if (deductibleState == 'relative') {
+            deductible = '\t\t\t<deductible isAbsolute="false"/>\n';
+            deductibleValue = ' deductible="'+data[i][deductibleInx]+'"';
+        }
+
+        // Retrofitted
+        var retrofittingSelect = $('#retrofittingSelect input:checked').val();
+        if (retrofittingSelect == 'retrofitting') {
+            retrofitting = 'retrofitted="'+data[i][retrofittingInx]+'"';
+        }
+
+        // limit value
+        var limitSelect = $('#limitSelect input:checked').val();
+        if (limitSelect == 'retrofitting') {
+            limit = 'insuranceLimit="'+data[i][limitInx]+'"';
+        }
+
+        // Deductible
+        if (true) {};
+
+        // Economic Cost
+        if (structuralInx > -1 ) {
+            costTypes += '\t\t\t\t<costType name="structural" type="per_asset" unit="USD" />\n';
+            costs += '\t\t\t\t\t<cost type="structural" value="'+ data[i][structuralInx]+'" '+retrofitting+' '+deductibleValue+' '+limitValue+'/>\n';
+        }
+        if (non_structuralInx > -1 ) {
+            costs += '\t\t\t\t\t<cost type="non_structural" value="'+ data[i][non_structuralInx]+'" '+deductibleValue+' '+limitValue+'/>\n';
+        }
+        if (contentsInx > -1 ) {
+            costs += '\t\t\t\t\t<cost type="contents" value="'+ data[i][contentsInx]+'" '+deductibleValue+' '+limitValue+'/>\n';
+        }
+        if (businessInx > -1 ) {
+            costs += '\t\t\t\t\t<cost type="business" value="'+ data[i][businessInx]+'" '+deductibleValue+' '+limitValue+'/>\n';
+        }
+
+        // Occupancies
+        if (dayInx > -1 ) {
+            occupancies += '\t\t\t\t\t<occupancy occupants="'+ data[i][dayInx]+'" period="day"/>\n';
+        }
+        if (nightInx > -1 ) {
+            occupancies += '\t\t\t\t\t<occupancy occupants="'+ data[i][nightInx]+'" period="night"/>\n';
+        }
+        if (transitInx > -1 ) {
+            occupancies += '\t\t\t\t\t<occupancy occupants="'+ data[i][transitInx]+'" period="transit"/>\n';
+        }
+
+        costs += '\t\t\t\t</costs>\n';
+        occupancies += '\t\t\t\t</occupancies>\n';
+
+        asset +=
+            '\t\t\t<asset id="'+id+'" '+number+' '+area+' '+taxonomy+' > \n' +
+                '\t\t\t\t<location '+longitude+' '+latitude+' />\n' +
+                costs +
+                occupancies +
+            '\t\t\t</asset>\n';
+    }
+
+    // Create a NRML element
+    var NRML =
```

Comment:
I guess there must be a simpler way in javascript to produce the xml, given tags and attributes